src/plugins/dash/setup-text-tracks.js

```diff
@@ -64,6 +64,10 @@ function attachDashTextTracksToVideojs(player, tech, tracks) {
 
     // Add track to videojs track list
     .map(({ trackConfig, dashTrack }) => {
```

Comment:
Please consider replace map with reduce, so you don not need use filer and loop on array twice